docroot/themes/humsci/humsci_basic/src/scss/utilities/_fonts.scss

```diff
@@ -42,3 +42,10 @@
   @include hb-link--inline;
 }
 
+.hb-pill-item {
```

Comment:
Instead of two separate classes we should have a single `.hb-pill` class and add the link styles by targeting `a.hb-pill`.